apidoc/Titanium/Geolocation/Geolocation.yml

```diff
@@ -395,6 +428,8 @@ properties:
     type: Number
     permission: read-only
     platforms: [android, iphone, ipad]
+    deprecated:
+        since: "2.0.0"
```

Comment:
I don't think they are deprecated for iOS. Maybe only deprecate and remove for Android?